airbyte-commons-worker/src/main/java/io/airbyte/workers/WorkerMetricReporter.java

```diff
@@ -21,9 +22,9 @@ public WorkerMetricReporter(final MetricClient metricClient, final String docker
     this.metricClient = metricClient;
   }
 
-  public void trackSchemaValidationError(final String stream) {
+  public void trackSchemaValidationError(final AirbyteStreamNameNamespacePair stream) {
     metricClient.count(OssMetricsRegistry.NUM_SOURCE_STREAMS_WITH_RECORD_SCHEMA_VALIDATION_ERRORS, 1, new MetricAttribute("docker_repo", dockerRepo),
-        new MetricAttribute("docker_version", dockerVersion), new MetricAttribute("stream", stream));
+        new MetricAttribute("docker_version", dockerVersion), new MetricAttribute("stream", stream.toString()));
```

Comment:
This goes in the label on DD, I think it would make sense to have an explicit format here. Maybe something like `"$namespace.$name"` rather than the default toString which gives `AirbyteStreamNameNamespacePair{name=$name, namespace=$namespace}`?

Reply:
@gosusnp good catch! I've change this to snake case. This will be `namespace_name`. How is that?

Aside, is it useful to add this tag?

Reply:
I added a small suggestion further down, especially since `namespace` is optional.

I don't know if this tag is useful, I would expected it is probably too specific to be usable. I haven't had a chance to play with it to tell for sure.

Reply:
thanks!